controller/src/main/java/io/pravega/controller/server/ControllerServiceStarter.java

```diff
@@ -197,7 +197,8 @@ protected void startUp() {
                 // Create ControllerEventProcessor object.
                 controllerEventProcessors = new ControllerEventProcessors(host.getHostId(),
                         serviceConfig.getEventProcessorConfig().get(), localController, checkpointStore, streamStore,
-                        hostStore, segmentHelper, connectionFactory, streamMetadataTasks, controllerExecutor);
+                        hostStore, segmentHelper, connectionFactory, streamMetadataTasks, streamTransactionMetadataTasks,
```

Comment:
One clarification: why were we not passing `streamTransactionMetadataTasks` before?